### PR TITLE
Added procesing for native url formats of Yii2

### PR DESCRIPTION
**Pull request by githubjeka**
For expample I can't use now:
```php
 $I->sendAjaxPostRequest( [/*...*/], $dataPost); 
 ```

This fix allow to do this

### CODE REVIEW

File: src/Codeception/Module/Yii2.php [hunk 1]
```diff
@@ -480,6 +470,26 @@ public function amOnRoute($route, array $params = [])
...
+     * @param $method
+     * @param $uri
+     * @param array $parameters
+     * @param array $files
+     * @param array $server
+     * @param null $content
+     * @param bool $changeHistory
+     * @return mixed
+     */
+    protected function clientRequest($method, $uri, array $parameters = array(), array $files = array(), array $server = array(), $content = null, $changeHistory = true)
```
Comment (sergeyklay) [inline src/Codeception/Module/Yii2.php]: Use short syntax 

```php
protected function clientRequest($method, $uri, array $parameters = [], array $files = [], array $server = [], $content = null, $changeHistory = true)
```

we are in the 21st century

Comment (githubjeka) [inline src/Codeception/Module/Yii2.php]: I just copied

**Comment by githubjeka** [comment]
Have you any tests for the Yii2 module?

**Comment by DavertMik** [comment]
@githubjeka yes, we run test of yii-basic app

**Comment by githubjeka** [comment]
I see https://github.com/Codeception/Codeception/blob/2.2/.travis.yml#L143
Ok. [The test](https://github.com/yiisoft/yii2-app-basic/blob/master/tests/functional/LoginFormCest.php#L6) uses the new feature.

**Comment by DavertMik** [comment]
Thanks!

**Pull request merged**